File: python_password_generator/data_models.py
class Login:
    def __init__(self, site, username, password):
        self.site = site
        self.username = username
        self.password = password
        
    def to_dict(self):
        return {
            "site": self.site,
            "username": self.username,
            "password": self.password
        }

File: password_manager_electron/API/dataModels.cs
namespace API
{
    public class User
    {
        public string username { get; set; }
        public string password { get; set; }
        public string hashed_password { get; set; }
    }

    public class _Login
    {
        public string site { get; set; }
        public string username { get; set; }
        public string password { get; set; }
        public string hashed_password { get; set; }
    }
}
